setup.py

```diff
@@ -0,0 +1,33 @@
+# Always prefer setuptools over distutils
+from setuptools import setup, find_packages
+# To use a consistent encoding
+from codecs import open
+from os import path
+
+here = path.abspath(path.dirname(__file__))
```

Comment:
i'm like 99% sure that defining `here` and importing a special `open` doesn't do anything. Most likely whatever template or tutorial you were looking at were expecting you to do something like `long_description=open(path.join(here, 'README'))`.